doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md

```diff
@@ -195,11 +193,17 @@ The following code example illustrates the implementation of a user verifier des
 
 To provision your user verifier, do the following:
 
-1. Create and compile user verifier configuration schema.
-2. Create and compile user verifier implementation.
-3. Create and compile user verifier descriptor.
-4. Place the user verifier descriptor and configuration classes into the Admin UI class path. Usually it is ```/usr/lib/kaa-node/lib```.
-5. Place the user verifier implementation classes into the Operations Server class path (```/usr/lib/kaa-node/lib```).
-6. Use [Admin UI]() or [REST API]() to create/update/delete your user verifier instances.
+1. Create maven or gradle project.
```

Comment:
Please reuse this text with minor changes: https://github.com/kaaproject/kaa/pull/777/files#diff-93047e57907cbe91126a705b789cff07R116